app/models/game.rb

```diff
@@ -0,0 +1,8 @@
+class Game < ApplicationRecord
+  validates :name, presence: true,
+                   length: { minimum: 2 }
+  validates :min_players, presence: true,
+                          numericality: { greater_than: 0 }
+  validates :max_players, presence: true,
+                          numericality: { less_than: 100 }
```

Comment:
Like this it could be 0 or smaller then min_players.
I'm not sure if we can validate this in here.